lib/pbench/server/utils.py

```diff
@@ -88,3 +91,23 @@ def quarantine(dest, logger, *files):
                 'quarantine {} {!r}: "mv {} {}/" failed', dest, files, afile, dest
             )
             sys.exit(102)
+
+
+class IsoTimeHelper:
+    def __init__(self, time: datetime.datetime):
+        self.utc_time = time
+        if (
+            self.utc_time.tzinfo is None
+            or self.utc_time.tzinfo.utcoffset(self.utc_time) is None
+        ):
```

Comment:
Can this be [simplified](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcoffset) to just?
```
        if (self.utc_time.utcoffset() is None):
```